device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,100 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '46133248' %}
+{% set ingress_lossless_pool_xoff =  '46133248' %}
```

Comment:
The same.

Reply:
Thanks Stephen, I will incorporate these changes.